### DISCUSSION TITLE
No browsers on i9300

### DISCUSSION THREAD

**Discussion by kurt-by** [Divested-Mobile/DivestOS-Build#50] [Q&A]
Hi,
Just discovered: you still build samsung galaxy S3 i9300. Thank you for that. 
Sadly i can't run any browser. No preinstalled browser, bromite and fenec immdiately stop after tapping app icon.

**Comment by SkewedZeppelin** [comment]
This is a known issue.
Likely due to the limited RAM available combined with the hardening.

I had a user recently test some tweaks, and they sadly did not fix it.

**Reply by kurt-by** [reply]
Is there an git issue to track?
